src/screens/home/HomeScreen.tsx

```diff
@@ -109,8 +110,13 @@ const Content = ({isBottomSheetExpanded}: ContentProps) => {
       if (!network.isConnected) {
         return <NetworkDisabledView />;
       }
+
       return exposureStatus.needsSubmission ? (
-        <DiagnosedShareView isBottomSheetExpanded={isBottomSheetExpanded} />
+        exposureStatus.hasShared ? (
```

Comment:
Thinking either force hasShared to always be false for this PR or always default to the existing DiagnosedShareView for this PR so that we can merge to Master without breaking existing functionality (before the rest of the pieces for this flow are done).  

You can add back in a follow-up PR after.

Reply:
So the `DiagnosedShareUploadView` screen appears when `hasShared` is false (because use has not shared the upload)  so changing it to true will take user to the `DiagnosedShareView `, even if they have not finished uploading (instead of false which will do the opposite). That work?

Reply:
Yes that works but we don't want that to happen until the rest of the functionality for this new flow is complete.

If we can adjust and merge this PR and than re-introduce (hasShared) we will avoid the possibility of a user hitting the new screen in the meantime.